NetCoreDemos/Views/DropDownButton/Overview.cshtml

```diff
@@ -154,3 +168,76 @@
         }
     }
 </script>
+
+<style>
+    .demo-container .dx-fieldset:first-child {
+        width: 500px;
+    }
+
+    .custom-color-picker {
```

Comment:
i believe we alredy have this style. please check it. and why did not you use css file for this demo?